payments-core/src/main/java/com/stripe/android/GooglePayJsonFactory.kt

```diff
@@ -39,6 +47,16 @@ class GooglePayJsonFactory constructor(
         isJcbEnabled = isJcbEnabled
     )
 
+    @Inject
+    constructor(
```

Comment:
Can it be internal? Same for others in non-internal classes

Reply:
yes, done